plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/operation/ImportNewProject.java

```diff
@@ -155,4 +164,22 @@ protected boolean cloneDestinationExists() {
 		return cloneDestination != null
 				&& cloneDestination.exists();
 	}
+
+	private static Collection<String> sanitize(Collection<String> filters) {
+		Collection<String> sanitized = null;
+		if (filters != null) {
+			sanitized = new LinkedHashSet<>(filters.size());
+			for (String path : filters) {
+				if (!StringUtils.isEmptyOrNull(path)) {
```

Comment:
Yes, several StringUtils libs are available in the classpath. I chose the one from the project